src/Events/Notifications/Notifications.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Class that handles interfacing with TEC\Common\Notifications.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Notifications
+ */
+
+namespace TEC\Events\Notifications;
+
+use Tribe__Events__Main as TEC;
+
+/**
+ * Class Notifications
+ *
+ * @since   TBD
+ * @package TEC\Events\Notifications
+ */
+class Notifications {
+
+	/**
+	 * Determines if we are on a TEC admin page except the post edit page.
+	 *
+	 * @since TBD
+	 *
+	 * @return boolean
+	 */
+	public static function is_tec_admin_page(): bool {
+		$current_screen = get_current_screen();
+		$helper         = \Tribe__Admin__Helpers::instance();
+
+		// Are we on a tec post-type admin screen?
+		if ( ! $helper->is_post_type_screen( TEC::POSTTYPE ) ) {
+			return false;
+		}
+
+		// Are we on a new post screen?
+		if ( $current_screen instanceof \WP_Screen && $current_screen->action === 'add' ) {
```

Comment:
Prefer use statements. Its cleaner and makes it easy to understand all the classes being used in a file by just reading the top lines of the file.

`use Tribe__Admin__Helpers as Admin_Helpers;`
`use WP_Screen`